web-console/script/druid

```diff
@@ -73,10 +73,12 @@ function _build_distribution() {
   _log "Building druid distribution"
 
   (
+    # Add HEAD as an allowed HTTP method since this is how we check when the Druid service is ready.
```

Comment:
Can't you do a GET on /status/health?

Reply:
I think this is checking to see if the web console is up. I'm not sure if a `GET /status/health` is equivalent. The nice side-benefit of this is I get an integration test for free 😇